tests/longevity-mv-si-4days.yaml

```diff
@@ -1,18 +1,18 @@
-test_duration: 2880
-prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_profile_4mv_5queries.yaml ops'(insert=15,read1=1,read2=1,read3=1,read4=1,read5=1)' cl=QUORUM n=10000000 -port jmx=6868 -mode cql3 native -rate threads=10",
-                    "cassandra-stress user profile=/tmp/c-s_profile_3si_5queries.yaml ops'(insert=25,si_read1=1,si_read2=1,si_read3=1,si_read4=1,si_read5=1)' cl=QUORUM n=10000000 -port jmx=6868 -mode cql3 native -rate threads=10"
+test_duration: 5760
```

Comment:
IN this case we need to change the file name as well from 2days to 4 days.

Reply:
It's named "4days"